#goal definition
float64 distance
---
#result definition
float64 distanceCompleted
---
#feedback
float64 distanceCurrent
